Dockerfile

```diff
@@ -1,44 +1,16 @@
-FROM ubuntu:jammy-20220815 as build
-RUN apt-get update
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
-	    sudo \
-            wget \
-            apt-transport-https \
-            dotnet-sdk-6.0 \
-            git \
-            cmake \
-            build-essential \
-            libssl-dev \
-            pkg-config \
-            libboost-all-dev \
-            libsodium-dev \
-            libzmq3-dev \
-            libzmq5
-
-
+FROM mcr.microsoft.com/dotnet/sdk:6.0-jammy as BUILDER
 WORKDIR /app
-RUN mkdir /app/build
+RUN apt-get update && \
+    apt-get -y install cmake build-essential libssl-dev pkg-config libboost-all-dev libsodium-dev libzmq5 libzmq3-dev golang-go
```

Comment:
`golang-go` isn't required,

Reply:
Do we want to assume everyone has buildx setup?  Because if you set the caching option on the apt installs it makes the build blazing fast.

Reply:
not sure. But in my experience the `dotnet` step is the one that takes the most time.